pkg/options/websocket.yaml

```diff
@@ -0,0 +1,19 @@
+imports:
+  - time
+
+docs:
+  title: Websocket server
```

Comment:
Can we be a bit more verbose? Besides title, you can add an intro that's used for the documentation gen.

Reply:
Yes